packages/lib/file_ops.js

```diff
@@ -68,7 +68,25 @@ async function findUnusedName(directory, name, extension = "") {
 		}
 	}
 }
-
+/*
+ * Warning: this function should not be relied upon to be accurate for
+ * security sensitive applications.  The selection process is inherently
+ * racy and a file or folder may have been created in the folder by the time
+ * this function returns.
+ *
+ * @param {string} prefix - Prefix for file
+ * @param {string} suffix - Suffix for file
+ * @param {string} tmpdir - Directory for temp file
+*/
```

Comment:
The reason the linter is angry is because its a bit picky about the formatting. You want 

```
/**
 *
 */
```

note the double star and spaces before the stars.